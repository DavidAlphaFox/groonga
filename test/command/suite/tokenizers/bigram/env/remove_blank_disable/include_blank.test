#$GRN_NGRAM_TOKENIZER_REMOVE_BLANK_DISABLE=yes

tokenize TokenBigramSplitSymbolAlpha "Hong Kong" NormalizerAuto --mode GET
