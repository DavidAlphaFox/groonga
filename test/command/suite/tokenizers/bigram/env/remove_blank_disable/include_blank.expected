tokenize TokenBigramSplitSymbolAlpha "Hong Kong" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "ho",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "on",
      "position": 1,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "ng",
      "position": 2,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "g ",
      "position": 3,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": " k",
      "position": 4,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "ko",
      "position": 5,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "on",
      "position": 6,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "ng",
      "position": 7,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
